astropy/cosmology/tests/test_realizations.py

```diff
@@ -38,30 +38,28 @@ class Test_default_cosmology(object):
     # -----------------------------------------------------
     # Get
 
+    def test_get_current(self):
+        """Test :meth:`astropy.cosmology.default_cosmology.get` current value."""
+        cosmo = default_cosmology.get()
+        assert cosmo is default_cosmology.validate(default_cosmology._value)
+
+    # -----------------------------------------------------
+    # _get_from_registry
+
     def test_get_fail(self):
         """Test bad inputs to :meth:`astropy.cosmology.default_cosmology.get`."""
         # a not-valid option, but still a str
         with pytest.raises(ValueError, match="Unknown cosmology"):
-            default_cosmology.get("fail!")
+            default_cosmology._get_from_registry("fail!")
```

Comment:
I don't think we should test private methods. Replace with `.validate`? Or remove?

Reply:
I'll replace with ``.validate``. 👍